priv/repo/tenant_migrations/20240213214201_prefer_email_over_auth0_uid.exs

```diff
@@ -28,7 +28,7 @@ defmodule BelayBrokerage.Repo.Migrations.PreferEmailOverAuth0Uid do
     for %{id: auth0_uid, email: email} <- investors do
       new_investor_id = Ecto.UUID.autogenerate()
 
-      repo().query!("UPDATE #{prefix()}.investors SET id = '#{new_investor_id}';")
+      repo().query!("UPDATE #{prefix()}.investors SET id = '#{new_investor_id}' WHERE id = '#{auth0_uid}';")
       repo().query!("INSERT INTO #{prefix()}.auth_accounts (uid, investor_id, email, is_primary) VALUES ('#{auth0_uid}', '#{new_investor_id}', '#{email}', true);")
```

Comment:
We need a where clause on this too

Reply:
@ktayah 
`auth_accounts` doesn't have any data in it yet.

Reply:
Right it's an insert, doh